automation/src/test/resources/test-grid-is-resources/SolutionPattern22/Tests/scenario-results.jtl

```diff
@@ -0,0 +1,3988 @@
+            } else if (&apos;client_credentials&apos; == grantType) {
+                document.getElementById(&quot;clientsecret&quot;).style.display = &quot;&quot;;
+                document.getElementById(&quot;callbackurltr&quot;).style.display = &quot;none&quot;;
+                document.getElementById(&quot;authzep&quot;).style.display = &quot;none&quot;;
+                document.getElementById(&quot;accessep&quot;).style.display = &quot;&quot;;
+                document.getElementById(&quot;recownertr&quot;).style.display = &quot;none&quot;;
+                document.getElementById(&quot;recpasswordtr&quot;).style.display = &quot;none&quot;;
+                document.getElementById(&quot;formPost&quot;).style.display = &quot;none&quot;;
+            }
+
+            return true;
+        }
+
+        function getAcceesToken() {
+            var fragment = window.location.hash.substring(1);
+            if (fragment.indexOf(&quot;&amp;&quot;) &gt; 0) {
+                var arrParams = fragment.split(&quot;&amp;&quot;);
+
+                var i = 0;
+                for (i = 0; i &lt; arrParams.length; i++) {
+                    var sParam = arrParams[i].split(&quot;=&quot;);
+
+                    if (sParam[0] == &quot;access_token&quot;) {
+                        return sParam[1];
+                    }
+                }
+            }
+            return &quot;&quot;;
+        }
+
+        function getIDtoken() {
+            var fragment = window.location.hash.substring(1);
+            var arrParams = fragment.split(&quot;&amp;&quot;);
+            for (var i = 0; i &lt; arrParams.length; i++) {
+                var urlParameters = arrParams[i].split(&quot;=&quot;);
+
+                if (urlParameters[0] == &quot;id_token&quot;) {
+                    var idToken = urlParameters[1];
+                    return idToken;
+                }
+            }
+            return &quot;&quot;;
+        }
+
+        function getDecodedIDToken() {
+            var idToken = getIDtoken();
+            if (idToken) {
+                var decodedIdToken = atob(idToken.split(&quot;.&quot;)[1]);
+                return decodedIdToken;
+            }
+            return &quot;&quot;;
+        }
+
+        function makeList(data) {
+            document.write(&apos;&lt;tbody&gt;&apos;);
+            for (var i in data) {
+                document.write(&apos;&lt;div&gt;&lt;tr&gt;&lt;td&gt;&lt;label id=&quot;idtokenList&quot;&gt;&apos;)
+                document.write(i);
+                document.write(&apos;&lt;/label&gt;&lt;/td&gt;&lt;td&gt;&apos;);
+                document.write(data[i]);
+                document.write(&apos;&lt;/td&gt;&lt;/tr&gt;&lt;/div&gt;&apos;);
+            }
+            document.write(&apos;&lt;/tbody&gt;&apos;);
+        }
+
+    &lt;/script&gt;
+
+&lt;/head&gt;
+&lt;!-- ===================================== END HEADER ===================================== --&gt;
+&lt;body&gt;&lt;a id=&quot;top-of-page&
```

Comment:
The JTL is 4000 lines it seems.. :) I don't think we need one this big for unit tests.

Reply:
New issue [1] created.

[1]. https://github.com/wso2/testgrid/issues/734